### DISCUSSION TITLE
Some dependency packages are missing in the created pythonRequirements.zip file

### DISCUSSION THREAD

**Discussion by kaojunsong** [UnitedIncome/serverless-python-requirements#844] [Q&A]
I am using 5.1.1 version, there is a quite strange issue happens in one of our environments: some Python packages are missing in the pythonRequirements.zip file.
Can anybody tell me how I can debug such kind of issue?

**Comment by pgrzesik** [comment]
Hey @kaojunsong 👋 That is indeed very strange behavior. Are you able to share a minimal reproducible example? Maybe I could try to test it out and debug on my side
